src/components/v2v3/V2V3Project/V2V3PayButton/V2V3ConfirmPayModal/V2V3PayForm.tsx

```diff
@@ -33,22 +45,29 @@ export interface V2V3PayFormType {
 }
 
 export const V2V3PayForm = ({
```

Comment:
previously this component used the now deleted SummaryTable component to help organize things. after moving the beneficiary component into what was previously the SummaryTable, it was too complex to operate the form across two different components, so i've just moved everything from SummaryTable into this component.